site/src/components/diagrams/Diagrams.module.css

```diff
@@ -62,3 +62,17 @@
 .textContainer {
   flex: 1;
 }
+
+.figure {
+  display: flex;
+  flex-direction: column;
+  gap: var(--salt-spacing-100);
+  margin: 0;
+}
+
+.caption {
+  font-style: var(--salt-editable-help-fontStyle);
```

Comment:
highlighted to paul moving this token to text characteristic for theme 2, as usage like here doesnt really make sense as 'editable'